Implement Booking methods

FernoCosti opened the pull request:
- Impelment get all bookings made by one customer.
- Implement get all bookings made on a certain travelpackage
- Implement get all bookings made to a certain destination

FernoCosti committed:
Implement Booking methods

Adrianpush reviewed (approved):
No need for destination class - Destination will be a string
In Booking Controller - new get mappings should receive parameters for customer ID, travel package ID and destination
In Booking Service - Since Destination should not be a class, get bookingbyDestination should have a String as parameter
In Booking Service Impl - Functions should not create new entities

Adrianpush committed:
Added
- Booking Controller (getBookingsByCustomer, getBookingByTravelPackage, getBookingsByDestination)
- Implemented methods in BookingServiceImpl
- Method for retrieving all TravelPackages by destination in Travel Package Repository.
